api/v1alpha1/backstage_types.go

```diff
@@ -31,14 +31,26 @@ type BackstageSpec struct {
 	// Raw Runtime Objects configuration. For Advanced scenarios.
 	RawRuntimeConfig RuntimeConfig `json:"rawRuntimeConfig,omitempty"`
 
+	// Configuration for database access. Optional.
+	DatabaseConfig DatabaseConfig `json:"databaseConfig,omitempty"`
```

Comment:
I know that naming things is one of hardest things in computing ^^, but how about just `database`? To be consistent with `application` (and make it understandable what the components of a default Backstage instance are)

Reply:
Yes that's good.